CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.1]
+
+-   Makes it so the editor adds a space after user and issue autocomplete suggestions.
+
 ## [0.4.2]
```

Comment:
Wasn't sure what version to use here. Looks like the latest version released is `0.5.0`. Not sure if this should be considered a fix or improvement.